### ISSUE TITLE
[Bug]: mo crashed when execute `drop table`

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#5927]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 0f3df071ca4b2b72f11b50d9197adcdef3905e81
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

1. on the client side, hang up after send `drop table xxx if existed`;
2. on mo-server side, mo crashed after a while, log content before the crash is as follows：
```
{"level":"INFO","time":"2022/10/24 07:55:15.749185 +0000","caller":"logservicedriver/appender.go:67","msg":"size is 157"}
panic: internal error: driver info: retry time out

goroutine 2494596 [running]:
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*driverAppender).append(0xc10b83ba00, 0x27d8c20?, 0x3b9aca00)
	/home/dan/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/appender.go:68 +0x4e6
created by github.com/matrixorigin/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver.(*LogServiceDriver).onAppendQueue
	/home/dan/matrixone/pkg/vm/engine/tae/logstore/driver/logservicedriver/append.go:66 +0xf2
```

full log: 
[aws-us-west-2-serverless-5ff57cc74-wtwfg.log.zip](https://github.com/matrixorigin/matrixone/files/9856669/aws-us-west-2-serverless-5ff57cc74-wtwfg.log.zip)

### Steps to Reproduce

```Markdown
1. start mo with the following command:

./mo-service -launch ./etc/launch-tae-logservice/launch.toml
```

2. execute some sqls
```


### Additional information

_No response_

**Comment by volgariver6** [comment]
cannot repro locally, need more details.

**Comment by volgariver6** [comment]
please add some details to help me repro, e.g. restart or load data? @sukki37

**Comment by volgariver6** [comment]
still cannot repro.

**Comment by volgariver6** [comment]
the key point of this problem is not "drop table xxx", it is becaue after long time running, heartbeat of cn or dn is not correct and causing logserivce cannot work well.

#6492 and #6495 may have resovled this problem. If there is still panic, contact me plz.

**Comment by aressu1985** [comment]
it seems that error message is the same as #6238, and has been fixed

fixed:e23e00f1d01d43e33c509a6d239ccc38b5f93409

this can be closed
